docs/api/v3.rst

```diff
@@ -164,6 +192,17 @@ Projects list
     The ``results`` in response is an array of project data,
     which is same as :http:get:`/api/v3/projects/(string:project_slug)/`.
 
+    .. note::
+
+       .. FIXME: we can't use :query string: here because it doesn't render properly
```

Comment:
I guess this is because only top elements are rendered, you could solve this by creating another subtitle `**Additional resources**` or something like that instead of a note.

Reply:
I think it's OK to use a note here to keep the same pattern with the rest of notes in the page to communicate some differences for commercial.

The FIXME is about not being able to use `:query string value:` and getting the rendering automatically. However, I solved that by manually "emulating" it (writing the same content with the same styles that it generates)